### PR TITLE
refactor(titleize): FRMW-397 convert titleize to a Quark filter

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-397

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `rxMisc.titleize` to Quark filter.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `titleize` Quark Filter

### Demo Page
screen shot 2015-12-18 at 10 53 09 am

### API Docs
screen shot 2015-12-18 at 10 53 34 am

**Comment by glynnis** [comment]
LGTM

File: src/quarks/titleize/docs/examples/titleize.simple.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div ng-controller="titleizeSimpleCtrl">
+    {{sample | titleize}}
```
Comment (CITguy) [inline src/quarks/titleize/docs/examples/titleize.simple.html]: reduce indentation to 2 spaces, per documented standards

**Comment by jgonz0010** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**